packages/gitmoji-changelog-cli/src/cli.spec.js

```diff
@@ -1,21 +1,26 @@
-const { changelog, logger } = require('@gitmoji-changelog/core')
+const { generateChangelog, logger } = require('@gitmoji-changelog/core')
 const { main } = require('./cli')
 
 describe('cli', () => {
-  it('should throw an error if not a git project', async () => {
-    changelog.mockImplementationOnce(() => {
+  it('should throw an error if changelog generation fails', async () => {
+    generateChangelog.mockImplementationOnce(() => {
```

Comment:
We could use [mockRejectedValue](https://jestjs.io/docs/en/mock-function-api#mockfnmockrejectedvaluevalue) :+1: